JShellAPI/README.MD

```diff
@@ -101,4 +101,97 @@ The maximum ram allocated per container, in megabytes.
 ### jshellapi.dockerCPUsUsage
 The cpu configuration of each container, see [--cpus option of docker](https://docs.docker.com/config/containers/resource_constraints/#cpu).
 ### jshellapi.schedulerSessionKillScanRate
-The rate at which the session killer will check and delete session, in seconds, see [Session timeout](#Session-timeout).
\ No newline at end of file
+The rate at which the session killer will check and delete session, in seconds, see [Session timeout](#Session-timeout).
+
+## Testing
+
+>  The work on testing was made in collaboration with [Alathreon](https://github.com/Alathreon) and [Wazei](https://github.com/tj-wazei). I'd like thank both of them for their trust. - FirasRG
```

Comment:
There is no need to say that, readme is supposed to rapidly indicate how to use this project.